admin_manual/maintenance/manually-moving-data-folders.rst

```diff
@@ -0,0 +1,91 @@
+=====================================
+How To Manually Move a Data Directory
+=====================================
+
+If you need to move your ownCloud data directory from its current location to
+somewhere else, here is a manual process that you can take to make it happen.
+
+.. NOTE:: 
+   This example assumes that:
+
+   - The current folder is: `/var/www/owncloud/data`
+   - The new folder is: `/mnt/owncloud`
+   - You're using Apache as your webserver
+
+1. Stop Apache
+2. Use rsync to sync the files from the current folder to the new one 
+3. Create a symbolic link from the new directory to the old one 
+4. Double-check `the directory permissions`_ on the new directory 
+5. Restart Apache
+
+To save time, here's the commands which you can copy and use::
+
+  apachectl -k stop 
+  rsync -avz /var/www/owncloud/data /mnt/owncloud
+  ln -s /mnt/owncloud /www/owncloud/data
+  apachectl -k graceful 
+
+.. NOTE:: 
+   If you're on Centos, try `service httpd stop` and `service httpd start`.
+   If you're on Debian/Ubuntu try `sudo service apache2 stop` and `sudo service apache2 start`
```

Comment:
Comments for lines 29-39:
1. `service $service $action` -> `systemctl $action $service` (`service` is deprecated)
2. `systemctl stop $service` -> `systemctl restart $service` (`systemctl` has the restart command, while service does not)